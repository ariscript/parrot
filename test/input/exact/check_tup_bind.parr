def foo(): (1, 2, 3)

check:
    example (a, b, c) = foo(),
    a is 1,
    b is== 2,
    c satisfies (lambda (x): x == 3).

nil
