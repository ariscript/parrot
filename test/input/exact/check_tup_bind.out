check block: 3/3 tests passed
nil
